front/src/components/login/LoginForm.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import { LoginButton, StyledCol, ForgotPassword, ErrorText} from './styledLoginForm';
+import { Form, Input, Col, Typography, Row } from 'antd';
+import 'antd/dist/antd.css';
+
+const { Title } = Typography;
+const layout = { 
+    labelCol: { span: 8, },
+    wrapperCol: { span: 16, },
+};
+
+export class LoginForm extends Component {
+
+    async validatePassword(_, value) {
+        const passwordRegex = /[a-zA-Z\d]{6,}$/;
+    
+        if (passwordRegex.test(value)) {
+          return Promise.resolve();
+        } else {
+          return Promise.reject('Password must be longer than 6 symbols');
+        }
+    }
+
+    async validateEmail(_, value) {
+        const emailRegex = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+    
+        if (emailRegex.test(value)) {
+          return Promise.resolve();
+        } else {
+          return Promise.reject('Email format must be like example@gmail.com');
+        }
+    }
+
+        
+    render() {
+        return (
+            <Row>
+                <StyledCol span={8} offset={8}>
+                    <Title level={2} align="center">Login</Title>
+
+                    <Form {...layout} name="basic" onFinish={this.props.onHandleLogin}>
```

Comment:
Use props destructuring at the beginning of the `render` method

Reply:
done